Makefile

```diff
@@ -99,15 +94,20 @@ third_party/libusb/webport/build/tests: third_party/googletest/webport/build
 
 endif
 
-# Enable JS-to-C++ tests in relevant configurations.
+# Enable JS targets in relevant configurations.
 ifneq (,$(findstring $(TOOLCHAIN),pnacl emscripten))
```

Comment:
Should this part (line 74)
```
# Applications are only built in non-sanitizer builds.
ifneq ($(TOOLCHAIN),asan_testing)
```
also be put here under (pnacl || emscripten) buildflag? I assume that applications don't need to be build for `coverage`.

Reply:
Good idea! Unfortunately the effect will be partially neglected by the fact that some test targets depend on the application targets (which btw doesn't look really necessary). But some overhead will go away already now. Thanks.

Reply:
Actually I forgot that there was an actual reason why we build the applications: the coverage script looks at the application binaries as well when calculating the resulting percentage. I've added a comment about that now.

But we can still do this small optimization for JS-only applications.

Reply:
Thanks!